src/lib/cht-session.ts

```diff
@@ -11,6 +11,7 @@ import { RemotePlace } from './remote-place-cache';
 const COUCH_AUTH_COOKIE_NAME = 'AuthSession=';
 const ADMIN_FACILITY_ID = '*';
 const ADMIN_ROLES = ['admin', '_admin'];
+const AUTHORIZED_ROLES = [...ADMIN_ROLES, 'user_manager'];
```

Comment:
The `user_manager` role is a part of cht app settings and thus configurable. Maybe we can make this dynamic by loading the configured role from the UMT config?

Reply:
Thanks for the idea. When you say UMT config, which file is that I take a look ?

Reply:
eg https://github.com/medic/cht-user-management/blob/main/src/config/chis-ke/config.json

Reply:
Or ask the CHT for the user's permissions and see if they have the permissions that are actually required?